src/include/ntp-client/dialogs.rb

```diff
@@ -124,12 +123,12 @@ def SimpleDialog
       Wizard.SetAbortButton(:abort, Label.CancelButton)
 
       startInit(nil)
-      CWM.handleWidgets(w, { "ID" => "never" })
+      CWM.handleWidgets(w, "ID" => "never")
 
       ret = CWM.Run(
         w,
         # yes-no popup
-        { :abort => fun_ref(method(:reallyExitSimple), "boolean ()") }
+        :abort => fun_ref(method(:reallyExitSimple), "boolean ()")
```

Comment:
To quick answer, my assumption was absolutely wrong, as also all rules are inherit from /usr/share/YaST2/data/devtools/data/rubocop_yast_style.yml, yeah, some rules are only applied for new code under src/lib